examples/flakes/README.md

```diff
@@ -0,0 +1,28 @@
+# Flakes
+
+Examples that show how to add custom flakes to your devbox project.
+
+# Local flakes (usually committed to your project)
+
+In devbox.json use "path:/path/to/flake#output" as the package name.
```

Comment:
Are these regular flake references? If so, could we call that out in case the user is already familiar with them?

Reply:
Yep, they are exactly what you use when installing a package that is a flake output. That is why `nix profile install` works almost as is with this change.